src/aliases.js

```diff
@@ -0,0 +1,31 @@
+import { promises as fs } from 'fs'
+import { version } from 'process'
+
+import findUp from 'find-up'
+
+export const NODE_VERSION_ALIAS = '.'
+export const NODE_VERSION_FILES = ['.node-version', '.nvmrc', '.naverc']
+
+export const CURRENT_NODE_ALIAS = '_'
+
+export const resolveVersionRangeAlias = async function (
+  versionRange,
+  { cwd } = {},
+) {
+  if (versionRange === CURRENT_NODE_ALIAS) return version
+
+  if (versionRange !== NODE_VERSION_ALIAS) return versionRange
+
+  const nodeVersionFile = await findUp(NODE_VERSION_FILES, { cwd })
+
+  if (!nodeVersionFile) {
+    throw new Error(
+      `node config file not found [was looking for ${NODE_VERSION_FILES.join(
```

Comment:
When no `NODE_VERSION_FILES`, we should default to `process.version`.

Reply:
I did. it, but I'm a bit unsure about it.

That is a good default, but it should be disableable. Otherwise no way to know if version was resolved from version file, or if just process version.

Reply:
I think this is an interesting idea, but I would favor simplicity (no options) over feature here as a first step. However if someone post an issue where they wished that an error was thrown to notify them that they forgot to add their `.nvmrc`, we should add that option then.